### PR TITLE
Modernize texts

### PR DESCRIPTION
**Pull request by robert-scheck**
…there is no more ICQ, MSN or AIM support in BitlBee

### CODE REVIEW

**Review by jelmer — CHANGES_REQUESTED** [review]

File: doc/user-guide/Installation.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@ The latest BitlBee release is always available from <ulink
 url="https://www.bitlbee.org/">https://www.bitlbee.org/</ulink>.
 Download the package with your favorite program and unpack it: <command>tar
 xvfz bitlbee-&lt;version&gt;.tar.gz</command> where &lt;version&gt; is to be
-replaced by the version number of the BitlBee you downloaded (e.g. 0.91).
+replaced by the version number of the BitlBee you downloaded (e.g. 3.6).
```
[the remark this answers is not in the document]
Comment (robert-scheck) [reply doc/user-guide/Installation.xml:13]: I know, but 0.91 is _very_ old now. So I thought a recent version makes sense for the next 10+ years.

File: doc/README [hunk 2]
```diff
@@ -112,8 +112,9 @@ BACKGROUNDS
 ===========
 
 We are both console lovers. But it is annoying to have a few tty's open with
```
[the remark this answers is not in the document]
Comment (robert-scheck) [reply doc/README]: Given I changed this…is the new variant fine? Or would it be more acceptable if I skip this part completely?

[the remark this answers is not in the document]
Comment (jelmer) [reply doc/README]: I'll defer to @Wilm0r as the original author

**Comment by robert-scheck** [comment]
@Wilm0r, kind ping?

**Comment by Wilm0r** [comment]
Yep, all seems reasonable to me, sorry for the delay!

**Pull request merged**